happi/client.py

```diff
@@ -325,6 +326,84 @@ def load_device(self, use_cache=True, **post):
         cntr = self.find_device(**post)
         return from_container(cntr, use_cache=use_cache)
 
+    def change_container(
+        self,
+        item: HappiItem,
+        target: Type[HappiItem],
+        edits: Dict[str, Any] = {},
+        how: str = 'right'
+    ) -> Dict[str, Any]:
+        """
+        Takes an instance of one item (device) and transfers its contents
```

Comment:
I don't think this docstring is true. The original item is untouched and no new item is created.